### PR TITLE
(tlg007 corpus) Plutarch Moralia 2nd batch

### PR DESCRIPTION
**Pull request by helmadik**
Mostly OCR issues again. Note that the word ᾖ was often missing, and 5 in the running text can stand for ὁ.

### CODE REVIEW

File: data/tlg0007/tlg079/tlg0007.tlg079.perseus-grc2.xml [hunk 1]
```diff
@@ -415,7 +415,7 @@
...
 <milestone unit="tlnum" n="5"/> δʼ ἄλλα θειότερα τούτων ἐκπλήττοντα καὶ κηλοῦντα τοὺς πολλοὺς διηγεῖσθαι , πάντων δὲ πίστιν ἔργῳ 
-<milestone unit="stephpage" xml:id="stephpage-163d" n="163d"/> παρασχεῖν. κύματος γὰρ ἠλιβάτου περὶ τὴν νῆσον αἰρομένου καὶ τῶν ἀνθρώπων δεδιότων ἀπαντῆσαι, μόνον ·ʼ ·ʼ<note resp="editor" place="unspecified" anchored="true" xml:lang="eng">lacunam signavit W. Fort. supplenda: <foreign xml:lang="grc">μόνῳ θαρρήσαντι τῷ Ἐνάλω προσελθεῖν τῇ θαλάττῃ</foreign> </note> θαλάττῃ ἕπεσθαι πολύποδας αὐτῷ πρὸς 
+<milestone unit="stephpage" xml:id="stephpage-163d" n="163d"/> παρασχεῖν. κύματος γὰρ ἠλιβάτου περὶ τὴν νῆσον αἰρομένου καὶ τῶν ἀνθρώπων δεδιότων ἀπαντῆσαι, μόνον <gap /><note resp="editor" place="unspecified" anchored="true" xml:lang="eng">lacunam signavit W. Fort. supplenda: <foreign xml:lang="grc">μόνῳ θαρρήσαντι τῷ Ἐνάλω προσελθεῖν τῇ θαλάττῃ</foreign> </note> θαλάττῃ ἕπεσθαι πολύποδας αὐτῷ πρὸς
```
Comment (lcerrato) [inline data/tlg0007/tlg079/tlg0007.tlg079.perseus-grc2.xml:418]: File failing compliance checks due to incomplete `<gap/>` tag.

**Comment by lcerrato** [comment]
tlg0007.tlg079.perseus-grc2.xml is not compliant

File: data/tlg0007/tlg079/tlg0007.tlg079.perseus-grc2.xml (hunk 1 above)
Comment (lcerrato) [reply data/tlg0007/tlg079/tlg0007.tlg079.perseus-grc2.xml:418]: should read `<gap reason="lost" rend="**"/>`

**Comment by helmadik** [comment]
Hi Lisa, this is a higher level of github knowledge.. Thanks so much for the correction; I am not sure this latest commit of the correction is in the right place, but am crossing my fingers! Many thanks.

**Comment by lcerrato** [comment]
@helmadik 
Looks good! I tried to change directly on here, but that didn't work as this came from your master repo, so I would have approved and then gone back to correct later.
With the changes to our tests, things will show as successful even when a previously compliant work is no longer passing, so we try to keep PRs to small batches and manually check the test results to avoid this.

**Review by lcerrato — APPROVED** [review]

**Comment by helmadik** [comment]
Woohoo! I'll get in touch offline to discuss how you'd prefer me to do things, and if all gap tags need rend attributes.

**Pull request merged**